integration/test/src/org/broadinstitute/reaper/server/IntegrationContainersTest.scala

```diff
@@ -0,0 +1,150 @@
+  ForAllTestContainer,
+  MultipleContainers,
+  PostgreSQLContainer
+}
+import com.google.auth.oauth2.{GoogleCredentials, ServiceAccountCredentials}
+import io.circe.Printer
+import io.circe.syntax._
+import org.broadinstitute.clio.client.webclient.ClioWebClient
+import org.broadinstitute.clio.integrationtest.ClioDockerComposeContainer
+import org.broadinstitute.clio.status.model.VersionInfo
+import org.broadinstitute.clio.util.auth.ClioCredentials
+import org.broadinstitute.clio.util.json.ModelAutoDerivation
+import org.junit.runner.Description
+import org.scalatest.{AsyncFlatSpecLike, Matchers}
+import slick.jdbc.JdbcBackend._
+
+import scala.collection.JavaConverters._
+
+class IntegrationContainersTest
+    extends TestKit(ActorSystem("reaper-int-test"))
+    with AsyncFlatSpecLike
+    with ForAllTestContainer
+    with ModelAutoDerivation
+    with Matchers {
+  lazy implicit val m: Materializer = ActorMaterializer()
+
+  lazy implicit val description: Description =
+    Description.createSuiteDescription(IntegrationContainersTest.super.getClass)
+
+  private val config: ReaperConfig = ReaperConfig
+    .load()
+    .fold(
+      failures => throw new RuntimeException(s"Could not load Reaper config: $failures"),
+      config => config
+    )
+
+  /** URL of vault server to use when getting bearer tokens for service accounts. */
+  private val vaultUrl = "https://clotho.broadinstitute.org:8200/"
+
+  /** Path in vault to the service account JSON to use in testing. */
+  private val vaultPath = "secret/dsde/gotc/test/clio/clio-account.json"
+
+  /** List of possible token-file locations, in order of preference. */
+  private val vaultTokenPaths = Seq(
+    File("/etc/vault-token-dsde"),
+    File(System.getProperty("user.home"), ".vault-token")
+  )
+
+  /** Pull service account credentials to use when accessing cloud resources from Vault. */
+  protected def loadCredentials(): GoogleCredentials = {
+    val vaultToken: String = vaultTokenPaths
+      .find(_.exists)
+      .map(_.contentAsString.stripLineEnd)
+      .getOrElse(sys.error("Vault token not found on filesystem!"))
+
+    val vaultConfig = new VaultConfig()
+      .address(vaultUrl)
+      .token(vaultToken)
+      .build()
+
+    val vaultDriver = new Vault(vaultConfig)
+    val accountJSON =
+      vaultDriver
+        .logical()
+        .read(vaultPath)
+        .getData
+        .asScala
+        .toMap[String, String]
+        .asJson
+
+    val jsonStream = new ByteArrayInputStream(
+      accountJSON.pretty(Printer.noSpaces.copy(dropNullValues = true)).getBytes()
+    )
+    ServiceAccountCredentials.fromStream(jsonStream)
+  }
+
+  private val reaperContainer = ReaperContainer.container
+  private val cromwellContainer = CromwellContainer.container
+  private val clioContainer: ClioDockerComposeContainer =
+    ClioDockerComposeContainer.waitForReadyLog(File("/tmp"))
+  private val postgresContainer = PostgreSQLContainer()
+
+  private lazy val db = Database.forURL(
```

Comment:
The database appears to be unused, is it required for reaper or for another component?

Reply:
Reaper doesn't use it yet but it was mentioned in the ticket that an sql db would be needed so I set up the container.